### ISSUE TITLE
Bot does not come online after login and start

### ISSUE THREAD

**Issue by StarTrekRules** [RogueException/Discord.Net#1273]
When i call LoginAsync and StartAsync nothing happens.
I have verified that the token is correct and that they are being called with a debugger, there are no errors or warnings in the console.

```cs
public static async Task Bot()
{
            DiscordSocketClient client = new DiscordSocketClient();

            await client.LoginAsync(Discord.TokenType.Bot, "-Token-");

            client.MessageReceived += OnMessage;

            await client.StartAsync();

            await Task.Delay(-1);
}

 public static void Main(string[] args)
 {
      try
      {
             Bot().Wait();
      }
      catch (Exception e)
      {
            // Error handling here
      }
}
```

**Comment by AtzederSerbe** [comment]
I had the same problem. For me it was that my bot had no permission to see any channel except for my welcome channel

**Comment by slemsvamp** [comment]
I have this problem as well, with a correct token. I made a new Core 2+ project and it worked fine, but I am trying to add it to my bot which is not running Core .NET, and even if I target 4.7.1 .NET Framework the LoginAsync just executes and never returns any value. No timeout, no errors, no complete task. It just doesn't do anything beyond logging the "Discord.Net v.2.0.1 (API v6)" after calling LoginAsync. I can't find anyone solving this problem except "I just changed to Core" which is not a valid solution for me. Can anyone confirm having this working on 4.7.1? If so, any hints?
